.editorconfig

```diff
@@ -481,9 +487,15 @@ dotnet_diagnostic.SA1142.severity = warning
 # Using directive should appear within a namespace declaration
 dotnet_diagnostic.SA1200.severity = none
 
+dotnet_diagnostic.SA1304.severity = none
+dotnet_diagnostic.SA1307.severity = none
+dotnet_diagnostic.SA1308.severity = none
```

Comment:
Why do we want to disable those (upper case for accessible fileds and no m_ and s_ prefixes)?

Reply:
Thank you for catching it!

I put SA1304 and SA1307 by adding new style section for `static read-only` fields. SA1308 and `s_` prefix style for static fields contradicts each other, so we need to keep that one off.